modules/cr_rich_text_paragraph/config/install/core.entity_form_display.paragraph.cr_rich_text_paragraph.default.yml

```diff
@@ -6,10 +6,10 @@ dependencies:
     - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_bg_colour
     - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_layout
     - field.field.paragraph.cr_rich_text_paragraph.field_body
-    - image.style.large
+    - image.style.medium
```

Comment:
is this change expected?

Reply:
I thought this was deliberate. It's the image style used for the preview image, so makes sense to use a smaller image style for that.

Reply:
As all of this config had been reverted/broken, I just used RND as reference point, and that uses the Medium preview size